youtube_callback/sftp_id_retrieve.py

```diff
@@ -48,21 +50,22 @@ def callfunction(course):
         shutil.rmtree(workdir)
     os.mkdir(workdir)
 
-    xml_downloader(course)
+    xml_csv_downloader(course)
 
     for file in os.listdir(workdir):
-        upload_data = domxml_parser(file)
+        if 'report-' in file:
```

Comment:
Why do we care whether the file name has `report-` in it or not? Do we have file that has other prefix? What do they do?

Reply:
The two files I'm seeing are "report-" and "error-". In the report file, if the video has an error status, there's an accompanying error file that contains the actual details of the error.

Reply:
I see. Thanks for the explanation